roles/core/nfs_client/molecule/default/molecule.yml

```diff
@@ -0,0 +1,26 @@
+dependency:
+  name: galaxy
+driver:
+  name: docker
+platforms:
+  - name: instance
+    groups:
+      - mg_computes
+    image: "quay.io/actatux/ansible-${MOLECULE_DISTRO:-centos:8}"
+    override_command: false
+    capabilities:
+      - "SYS_ADMIN"
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    privileged: true
+    pre_build_image: true
+provisioner:
+  name: ansible
+  ansible_args:
+    - -e mount_exported_dirs=false
+  inventory:
+    links:
+      group_vars: ../../../../../resources/examples/simple_cluster/inventory/group_vars
```

Comment:
Interesting usage of an example inventory. For molecule, I prefer not to rely on the examples inventory since they only cover a single use case, even if we can override any variable in the molecule files.
@oxedions @strus38 any comment on this?

Reply:
If we provides examples, why we do not use them ? 
I love this approach because It is cleaner to read.
Besides, using examples allow us to catch errors automatically for example if a variable name changes in `nfs.yml` file.

Reply:
This is interesting because it kind of validate also the examples. If we update the examples and "break" something, then all tests will fail.
Are we able to do one first "pass" with the example inventories (both ?) and then optionally do specific "passes" if we need to cover role specific use cases ?

Reply:
> using examples allow us to catch errors automatically for example if a variable name changes in nfs.yml file.
> [...]
> If we update the examples and "break" something, then all tests will fail.

This is exactly why I don't want to include these examples to the role tests. We are testing the roles, not the examples. Moreover, examples must remain examples. If we use these in the tests, they will become de facto standards.

> Are we able to do one first "pass" with the example inventories (both ?) and then optionally do specific "passes" if we need to cover role specific use cases ?

It might be possible to create molecule tests dedicated to the examples that would run the example playbooks along with the example inventories.

Reply:
>This is exactly why I don't want to include these examples to the role tests. We are testing the roles, not the examples. Moreover, examples must remain examples. If we use these in the tests, they will become de facto standards.

I can understand that. Then lets keep it the way it is.

Reply:
Ok guys, fair enough, I am taking in count this discussion, I will rework all the `molecule.yml` of my PR's.
Thank you !

Reply:
Inventory part revamped, commit : 4dd4ed5